ios/brave-ios/Sources/BraveUI/SwiftUI/BraveLoadingStyle.swift

```diff
@@ -0,0 +1,142 @@
+// Copyright 2025 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import DesignSystem
+import SwiftUI
+
+public struct BraveProgressStyleWave: ProgressViewStyle {
+  private var filledCircleColor = Color(UIColor(rgb: 0x7C91FF))
```

Comment:
this class doesn't seem to be used anywhere.

Reply:
I only moved it. It's from AI-Chat's design. Not used currently.